src/main/resources/templates/UpdateUserProfile.html

```diff
@@ -28,8 +28,37 @@
     <li><a href="/Settings">Settings</a></li>
     <li style="float:right"><form th:action="@{/logout}" method="post"><input type="submit" value="Log Out"/></li>
 </ul>
+<div class="row">
+    <div class="q-column">
+        <p>Current Username: [[${#httpServletRequest.remoteUser}]]</p>
+        <p>Current Email Address: </p>
+    </div>
+</div>
+<h2>Update Profile</h2>
+<div class="registrationcontainer">
+    <div class="row">
+        <div class="q-column">
+            <div class="logintext">
 
-
+                <input type="text" name="updateUsername"  placeholder="Update Username"><br>
+                <br>
+                <input type="password" name="updatePassword" placeholder="Update Password"><br>
+                <br>
+                <input type="email" name="updateEmail" placeholder="Update Email Address "><br>
+                <br>
+                <input type="text" name="Profession"  placeholder="Enter Profession (ex: Student, Server, etc.)"><br>
+                <br>
+                <input type="date" name="birthDate" placeholder="Enter Birth Date" required><br>
+                <br>
+                
```

Comment:
Looks good, but we don't want to allow users to be able to update their username and password on the homepage. Each user should be connected to one username and one password in order to connect them to their account.